File: evenSquare.go
package main
import (
	"fmt"
)

func square(number int) int {
	return number * number;
}

func main() {
	var n int;

	fmt.Scanln(&n);

	for i := 1; i < n + 1; i++ {
		if (i % 2 == 0) {
			fmt.Printf("%d^2 = %d\n", i, square(i));
		};
	};
};

File: evenSquare.rs
use std::io;

fn square(number: i32) -> i32 {
    return number * number
}

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: i32 = input.trim().parse().expect("Failed to parse input as integer");

    for idx in 1..n+1 {
        if (idx % 2 == 0) {
            println!("{}^2 = {}", idx, square(idx))
        }
    }
}